src/inc/gcinfotypes.h

```diff
@@ -376,12 +376,15 @@ enum infoHdrAdjustConstants {
     SET_PROLOGSIZE_MAX = 16,
     SET_EPILOGSIZE_MAX = 10,  // Change to 6
     SET_EPILOGCNT_MAX = 4,
-    SET_UNTRACKED_MAX = 3
+    SET_UNTRACKED_MAX = 3,
+    SET_RET_KIND_MAX = 4,   // 2 bits for ReturnKind
+    MORE_BYTES_TO_FOLLOW = 0x80 // If the High-bit of a header or adjustment byte 
+                               // is set, then there are more adjustments to follow.
```

Comment:
@briansull your comments showed up in my browser after I merged.
I'll make the change in a separate checkin.